osu.Game.Rulesets.Catch/UI/CatchPlayfield.cs

```diff
@@ -35,20 +37,15 @@ public class CatchPlayfield : ScrollingPlayfield
 
         public CatchPlayfield(BeatmapDifficulty difficulty)
         {
-            var droppedObjectContainer = new Container<CaughtObject>
-            {
-                RelativeSizeAxes = Axes.Both,
-            };
-
-            CatcherArea = new CatcherArea(droppedObjectContainer, difficulty)
+            CatcherArea = new CatcherArea(difficulty)
```

Comment:
With this change, can we now move this inline in the `InternalChildren` setter below? I think it was only done outside to allow for the convoluted setup order requirements.

Reply:
No, it has `CatcherArea.MovableCatcher.CreateProxiedContent(),` below.

Reply:
unfortunate